HdmiCec/HdmiCec.cpp

```diff
@@ -569,12 +578,27 @@ namespace WPEFramework
                 smConnection = NULL;
             }
             cecEnableStatus = false;
-
             if(1 == libcecInitStatus)
             {
-                LibCCEC::getInstance().term();
+                try
+                {
+                   LibCCEC::getInstance().term();
+                }
+                catch(InvalidStateException &e){
```

Comment:
if you printing with e.what() do ytou need to have separate case for exception?

Reply:
I think I see what you mean, I've updated to remove the specified case.